tarofreighter/chain_porter.go

```diff
@@ -399,7 +406,9 @@ func (p *ChainPorter) waitForPkgConfirmation(pkg *OutboundParcelDelta) {
 		},
 		Blob: updatedReceiverProof.Bytes(),
 	}
-	err = p.cfg.AssetProofs.ImportProofs(ctx, receiverProof)
+	err = p.cfg.AssetProofs.ImportProofs(
```

Comment:
I think this shows an issue in the current proof archive abstraction. I think we may want to move to a model where all the proofs are on disk and the db is just an index sooner than we anticipated. Alternatively, we can split the interface up a bit, into an importer and normal archive. The import would be used today in most places where we want to test node replication, and the archive behavior everywhere else. This would let us simplify this code a bit.